src/ert/cli/main.py

```diff
@@ -145,5 +145,4 @@ def run_cli(args: Namespace, plugin_manager: Optional[ErtPluginManager] = None)
 
     if end_event.failed:
         # If monitor has not reported, give some info if the job failed
-        msg = end_event.msg if args.disable_monitoring else ""
-        raise ErtCliError(msg)
```

Comment:
@yngve-sk / @oyvindeide  Is there a good reason for not propagating the error message here if monitoring is enabled?

Reply:
The monitor has already reported the error message, so it came out double